maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/data/builder/TestModelDataBuilder.java

```diff
@@ -1138,4 +1141,40 @@ public static UserSummaryDTO getUserSummaryDTO(List<String> roleActions, NewWork
                 .reservationsEntity(getReservationsDTO())
                 .build();
     }
+
+    public static uk.gov.justice.laa.crime.orchestration.dto.maat_api.FinancialAssessmentDTO getMaatApiFinancialAssessmentDTO() {
```

Comment:
Maybe this is an indication it's time to split up the `TestModelDataBuilder` class a little?

Reply:
Good idea. I do think it would be nice to have small builders for specific things, but it's not a major issue.